static/js/timerender.js

```diff
@@ -178,7 +178,10 @@ export function render_markdown_timestamp(time, text) {
     const timestring = format(time, "E, MMM d yyyy, " + hourformat);
     const titlestring = "This time is in your timezone. Original text was '" + text + "'.";
     return {
-        text: timestring,
+        text:
+            timestring +
+            " " +
+            time.toLocaleTimeString("en-us", {timeZoneName: "short"}).split(" ")[2],
```

Comment:
This would benefit from being a lot more readable and/or having tests,.

@YashRE42 can probably help with the testing part, by stacking it on top of his PR that adds tests for `rendered_markdown.js` logic.

For readability, `.split(" ")[2]` isn't great -- we should at least have a comment with an example, but better might be to use a function that just returns the value we want, most likely in the form of just adding an extra parameter to `format()` call if one exists.

Reply:
It looks like the right API is

```js
new Intl.DateTimeFormat([], {timeZoneName: "short"})
  .formatToParts(time)
  .find(({type}) => type === "timeZoneName").value
```